presidio-analyzer/pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "presidio_analyzer"
+description = "Presidio Analyzer package"
+license = {text = "MIT License"}
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+requires-python = ">= 3.5"
+keywords = ["presidio_analyzer"]
+urls = {Homepage = "https://github.com/Microsoft/presidio"}
+dependencies = [
+    "spacy>=3.4.4, <4.0.0",
+    "regex",
+    "tldextract",
+    "pyyaml",
+    "phonenumbers>=8.12,<9.0.0",
+]
+dynamic = ["version"]
+
+[project.readme]
+file = "README.md"
+content-type = "text/markdown"
+
+[project.optional-dependencies]
+transformers = ["spacy_huggingface_pipelines"]
+stanza = [
+    "stanza",
+    "spacy_stanza",
+]
+azure-ai-language = [
+    "azure-ai-textanalytics",
+    "azure-core",
+]
+test = ["pytest", "ruff"]
+
+[tool.setuptools.package-data]
+presidio_analyzer = [
+    "conf/*",
+]
+
+[tool.setuptools.dynamic]
+version = {file = ["VERSION"]}
```

Comment:
@SharonHart is there supposed a line that runs the app? something like 
[tool.flask]
app = "app:app"
?

Reply:
We run through docker, the old setup.py didnt have that line.